suites/pacific/rados/tier_3_rados.yaml

```diff
@@ -0,0 +1,50 @@
+tests:
+
+  # Cluster deployment stage
+
+  - test:
+      abort-on-fail: true
+      desc: Install software pre-requisites for cluster deployment.
+      module: install_prereq.py
+      name: setup pre-requisites
+
+  - test:
```

Comment:
I feel it's better to have all the various components like rgw, fs, rbd deployed on the cluster for executing rados tests.

Reply:
It is under discussion that do we need to maintain a global file or a separate file for each tier. According to that, I will implement.